package.json

```diff
@@ -141,5 +144,8 @@
     "yargs": "^3.0.4",
     "yiewd": "~0.6.0",
     "rewire": "^2.3.1"
+  },
+  "optionalDependencies": {
+    "udidetect": "^1.0.7"
```

Comment:
optionalDependencies get loaded along with all other dependencies, but if the install fails for any reason, installation resumes with just a warning.